dask/array/tests/test_array_function.py

```diff
@@ -34,6 +34,7 @@
         lambda x: np.round(x),
         lambda x: np.insert(x, 0, 3, axis=0),
         lambda x: np.delete(x, 0, axis=0),
+        lambda x: np.select([x < 0, x > 2, x > 1], [x, x * 2, x * 3], default=1),
```

Comment:
I know I suggested this test, but I just looked at it with fresh eyes and realized that it is very silly. It returns all 1...

How about this instead:

```suggestion
        lambda x: np.select([x < .5, x > .5], [x, -x], default=1),
```

Reply:
Yeap, but we still don't use the default value either overlap conditions. 
What do you think about something like this?

```python
np.select([x < .3, x < .6, x > .7], [x *2, x , x / 2], default=.65)
```

Reply:
sure!

Reply:
Done